OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -2038,6 +2050,9 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
                         if (second.TryGetValueAsString(out var s)) return s != "";
                         if (second.Value == null) return false;
                     }
+                    if (second.Value == null) {
```

Comment:
I forget why I put this instead of DreamValue.null

Reply:
Wait the above line means someone else added this as well and it was a bad merge methinks